code/game/objects/items/devices/flash.dm

```diff
@@ -170,7 +170,7 @@
 	//It will never break on the first use.
 	switch(times_used)
 		if(0 to 5)
-			if(prob(2*times_used))	//if you use it 5 times in a minute it has a 10% chance to break!
+			if(prob(10*times_used))	//More consequential rolls are made the more you overuse the device.

```

Comment:
This is an incredibly large jump. Have you tested this? According to the same math, 5 times in a minute = 50% chance to break (Although that math is dubious and I suspect it is even higher.)

Reply:
Basically, here's how it works.
flash first time: doesn't break. Guaranteed to blind.
second time: 10% to break on use.
third: 20%

And so on.